README.md

```diff
@@ -443,8 +443,10 @@ not enough to allow arbitrary package attacks against ECUs in the vehicle.
 The Director can only instruct clients to install images validated by the Image
 Repository.
 
-But what happens if, at the same time, the attacker is able to sign with an
-image-signing key for the Image Repository?
+But what happens if, at the same time, the attacker is able to get a hold of a
+high-level image-signing key trusted by the Image Repository? (Note that these
```

Comment:
This might  be nitpicking, but the previous phrasing captures the case where an attacker uses an HSM (containing the offline key) to maliciously sign for images.  The attacker doesn't necessarily need to get ahold of a high-level signing key.

Reply:
That is a good point. I don't usually worry about distinguishing those scenarios, but it's important not to confuse people. I'll switch the wording back to "sign with" in the incoming commit. (I could say "gains access" or "gains the use of", but I think "sign with" makes sense.)